src/Entity/Comment.php

```diff
@@ -130,7 +130,7 @@ public function getPost(): ?Post
         return $this->post;
     }
 
-    public function setPost(?Post $post): void
+    public function setPost(Post $post): void
```

Comment:
that breaks `Post::removeComment` (but the proper way there is not to set it as `null` here anyway, as the Comment will be removed entirely by the `orphanRemoval` feature, not be kept orphan)

Reply:
Thanks @stof, done.